utils/dashboard.js

```diff
@@ -28,77 +28,71 @@ const dashboard = async (req, res) => {
     let userEmail = SSOObject.email;
     let siteDetails = SSOObject.siteDetails;
 
-    // Auth success, and proceed below steps:
     const { isParentEntity } = require('./shared');
     const authObj = await isParentEntity(siteDetails);
-    if(userEmail) authObj['userEmail'] = userEmail;
+    if (userEmail) authObj['userEmail'] = userEmail;
     const isParent = authObj.isParent;
     const siteCodes = authObj.siteCodes;
     const isCoordinatingCenter = authObj.coordinatingCenter;
     const isHelpDesk = authObj.helpDesk;
     const api = req.query.api;
     console.log(`SMDB API: ${api}, accessed by: ${userEmail}`);
 
-    if(api === 'validateSiteUsers') {
-        const { validateSiteUsers } = require('./validation');
-        return await validateSiteUsers(req, res, authObj);
-    }
-    else if (api === 'getParticipants') {
+    if (api === 'validateSiteUsers') {
+        if (req.method !== 'GET') {
+            return res.status(405).json(getResponseJSON('Only GET requests are accepted!', 405));
+        }
+        return res.status(200).json({message: 'Ok', code: 200, isParent, coordinatingCenter: isCoordinatingCenter, helpDesk: isHelpDesk});
+    } else if (api === 'getParticipants') {
         const { getParticipants } = require('./submission');
         return await getParticipants(req, res, authObj);
-    }
-    else if (api === 'identifyParticipant' && isParent === false) {
+    } else if (api === 'identifyParticipant' && isParent === false) {
         const { identifyParticipant } = require('./submission');
         return await identifyParticipant(req, res, siteCodes);
-    }
-    else if (api === 'submitParticipantsData') {
+    } else if (api === 'submitParticipantsData') {
         const { submitParticipantsData } = require('./sites');
         return await submitParticipantsData(req, res, siteCodes);
-    }
-    else if (api === 'updateParticipantData') {
+    } else if (api === 'updateParticipantData') {
         const { updateParticipantData } = require('./sites');
         return await updateParticipantData(req, res, authObj);
-    }
-    else if (api === 'updateParticipantDataNotSite') {
-        if(req.method !== 'POST') {
+    } else if (api === 'updateParticipantDataNotSite') {
+        if (req.method !== 'POST') {
             return res.status(405).json(getResponseJSON('Only POST requests are accepted!', 405));
         }   
         const {submit} = require('./submission');
         let body = req.body;
-        if(!body.uid) {
+        if (!body.uid) {
```

Comment:
I know this is not your code, but there appears to be an error code discrepancy here (line 65).

Reply:
![Screenshot 2023-09-21 at 2 11 40 PM](https://github.com/episphere/connectFaas/assets/30497847/f64d3bf0-0f48-43c8-a4e1-c6c71da6fed8)

Reply:
Will fix it